string variable_name
---
byte[] data